specification/web/resource-manager/Microsoft.Web/stable/2023-12-01/examples/CreateOrUpdateFunctionApp.json

```diff
@@ -0,0 +1,378 @@
+{
+  "parameters": {
```

Comment:
This file seems to include all that can be possibly set for a function app regardless of SKU. Is that how it's meant to be? It'd be great to have a separate file specific for Flex Consumption that shows only what's posssible and won't be blocked for a FC app. For example: this sample has both LinuxFxVersion and a bunch of app settings, has the "Reserved" setting and others, etc. Not sure who looks at this sample but just calling it out

Reply:
I have updated the example to make it Flex Consumption specific. 

@davidmrdavid @nzthiago -- Could you please take another look? Thank you.